Bottom: 3ffe290f06e047c01845196527927cceaf08f84d
Top:    f1bc7c054451505a63b0c68d2a50cf73027c055c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-29 23:49:11 +0000

Refresh of benchmark-no-holes-dicer-perfo

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 3a9afe7..edfe2a7 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -1512,17 +1512,15 @@ ghid_draw_everything (BoxTypePtr drawn_area)
 
   memset (do_group, 0, sizeof (do_group));
   for (ngroups = 0, i = 0; i < max_layer; i++) {
-    LayerType *l;
     int group;
     int orderi;
 
     orderi = reverse_layers ? max_layer - i - 1 : i;
 
     // Draw in numerical order when in 3D view
-    l = global_view_2d ? LAYER_ON_STACK (i) : LAYER_PTR (orderi);
-    group = GetLayerGroupNumberByNumber (global_view_2d ? LayerStack[i] : orderi);
+    group = global_view_2d ? GetLayerGroupNumberByNumber (LayerStack[i]) : orderi;
 
-    if (/*l->On && */!do_group[group]) {
+    if (!do_group[group]) {
       do_group[group] = 1;
       drawn_groups[ngroups++] = group;
     }
